pages/_app.tsx

```diff
@@ -59,15 +61,11 @@ export default function App(props: AppProps) {
               </Navbar>
             }
             header={
-              <Header
-                height={80}
-                pt="xs"
-                pl="sm"
-                pb="xs"
-                style={{ backgroundColor: "#bdebf0", color: cyanMainShade }}
-              >
+              <Header height={80} style={{ backgroundColor: "#bdebf0", color: textGray }}>
                 <Center>
-                  <h2>DEQM Test Server Frontend</h2>
+                  <Link href={"/"}>
+                    <h1 style={{ marginTop: "12px" }}>DEQM Test Server Frontend</h1>
```

Comment:
```suggestion
                    <h1 style={{ marginTop: "12px", cursor: "pointer" }}>DEQM Test Server Frontend</h1>
```
This will turn the cursor to the Mickey Mouse glove on hover, showing the user that the text is clickable